panamaconverter/src/PanamaCIRCTConverter.scala

```diff
@@ -486,7 +484,7 @@ class PanamaCIRCTConverter(val circt: PanamaCIRCT, fos: Option[FirtoolOptions],
       def referToNewConstant(n: BigInt, w: Width, isSigned: Boolean): Reference.Value = {
         val (firWidth, valWidth) = w match {
           case _: UnknownWidth =>
-            val bitLen = util.bitLength(n)
+            val bitLen = if (!isSigned) max(n.bitLength, 1) else n.bitLength + 1
```

Comment:
plz add the documentation to this for explaining the reason.

Reply:
Done.